index.js

```diff
@@ -11,6 +12,7 @@ module.exports = {
         jquery: true,
         node: true
     },
+    parser: 'babel-eslint',
```

Comment:
Why was this necessary?

Reply:
Good question, this is mentioned in the readme [of the eslint plugin](https://github.com/salesforce/eslint-plugin-lwc#usage):

> Some of the syntax used in Lightning Web Components is not yet stage 4 (eg. decorators), and the out-of-the-box parser from ESLint doesn't support this syntax yet. In order to parse the LWC files properly, set the parser field to babel-eslint.

I noticed we had the parser set the same in Expensify.cash so I thought it was a harmless change, but maybe there are other side effects I didn't think about? Is there anything I should do to validate this won't break something else?

Reply:
Cool, thanks for the link and explanation. I am not worried about this breaking anything, I was more just curious about why it was necessary. Sometimes when trying to implement things like this, I end up trying out several things, and them some of them end up not being necessary and I forget to remove them. Just wanted to make sure this wasn't one of those!